builds/e2e/longhaul.yaml

```diff
@@ -13,6 +13,8 @@ variables:
   #pool.name: ''
   #container.registry: ''
   #snitch.storage.account: ''
+  #LogAnalyticEnabled
```

Comment:
```suggestion
  #log.analytic.enabled
```

Reply:
`LogAnalyticEnabled ` -> `logAnalytic.enabled`
`LogAnalyticLogType` -> `logAnalytic.logType`

Reply:
TODO: Once these changes are commit to remote, test them on the pipelines & roll them out beyond `Stress (Dev)` and `Long Haul` pipelines.

Reply:
Get both {Window AMD 64, LinuxAMD64} to run on both {Stress Test (Release Candidate), Long Haul (Release Candidate)}